src/components/RequestFormActions/RequestFormActions.tsx

```diff
@@ -43,15 +44,20 @@ export const RequestFormActions = ({
 
     const [comment, setComment] = useState<string | undefined>();
 
-    const { declineUserRequest, acceptUserRequest, cancelUserRequest, cancelTransferInternToStaffRequest } =
-        useUserCreationRequestMutations();
+    const {
+        declineUserRequest,
+        acceptUserRequest,
+        cancelUserRequest,
+        cancelTransferInternToStaffRequest,
+        cancelTransferInsideRequest,
+    } = useUserCreationRequestMutations();
     const { cancelScheduledDeactivation } = useScheduledDeactivation();
 
     const commentRef = useLatest(comment);
 
     const handleCancelSubmit = useCallback(() => {
         onCancel && onCancel();
-        if (requestType === 'deactivation') {
+        if (requestType === 'retirement' || requestType === 'transfer') {
```

Comment:
why are you using  string literlals here? but...

Reply:
this for deactivations, theirs types not in enum yet